BLSQ/iaso
# POLIO-812: HOTFIX link to campaign from lqas/im

When using the link to selected campaign from lqas/im, if the campaign is not on the table's first page, its data is not in the API response and the modal has no data to display

Related JIRA tickets : POLIO-812

## Self proof reading checklist


## Changes
- Add `search` param with the OBR name to the redirection url to force the API to return the required data

## How to test
Go to LQAS and IM , select a country then a campign, use the link to campaign button. A tab should open and the modal shoudl fetch the data.

## Print screen / video

https://user-images.githubusercontent.com/38907762/217040890-43982625-6fe9-43c3-a726-4e49bc7b9ec4.mov
